core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolverSpec.scala

```diff
@@ -370,5 +376,36 @@ class MethodConfigResolverSpec extends WordSpecLike with Matchers with TestDrive
         badTemplate.get
       }
     }
+
+    "resolve dos:// attributes and leave others alone" in {
+      val marthaDAO: MarthaDAO = (v) => Future.successful(s"gs://$v")
+      val values = Seq(AttributeString("dos://foo"), AttributeString("gs://bar"))
+      Await.result(MethodConfigResolver.mapDosToGs(values, marthaDAO), Duration.Inf) shouldBe Seq(AttributeString("gs://dos://foo"), AttributeString("gs://bar"))
+    }
+
+    "resolve dos:// URIs when evaluating input expressions" in {
```

Comment:
I really disliked how this test suite has a whole bunch of common fixtures and setup that aren't necessarily used by all test cases; it's too hard to tell which tests rely on which data. It also feels wrong to be testing individual cases of input expression evaluation at the same time as WDL parsing. This test localized everything and bypasses WDL parsing and is therefore both focused and self-contained.